syntax = "proto3";

message Point2{
    float x = 1;
    float y = 8;
    float z = 3;

}

message Numeric {
    float value = 1;
}

service ProcessingServices{
    rpc computeSum(Point2) returns (Numeric){}
}
